examples/spin-timer/Cargo.toml

```diff
@@ -9,7 +9,9 @@ anyhow = "1.0.68"
 clap = { version = "3.1.15", features = ["derive", "env"] }
 futures = "0.3.25"
 serde = "1.0.188"
+spin-cli = { path = "../.." }
```

Comment:
This is annoying have to bring the _very large_ `spin-cli` crate into here. The right thing to do is to move `RuntimeFactorsBuilder` impl out of `spin-cli` into its own crate. It's yet another crate which isn't ideal, but better than this alternative.

Reply:
I have refactored so that this is no longer the case.